### DISCUSSION TITLE
Has plan to support Deno or Bun?

### DISCUSSION THREAD

**Discussion by xbol0** [deta/cloud-docs#536] [Ideas] [answered]
Both they are awesome JS runtime.

**Comment by jnsougata** [comment, marked as answer]
Deta Space has already support for Bun

**Reply by CyberL1** [reply]
And what about Deno?

**Reply by jnsougata** [reply]
Ig that too.

**Reply by CyberL1** [reply]
I don't think so as there is nothing in the docs, even the "custom" one doesn't have it (says Deno not found) and there is no "unzip" too so I cannot even install it

**Reply by rareseu** [reply]
Hmm, can an app from Deta Space be deployed for public like Deta Micro?
